src/object/srv_obj_migrate.c

```diff
@@ -3303,8 +3312,8 @@ migrate_obj_ult(void *data)
 		DP_RB_MPT(tls), DP_UOID(arg->oid), arg->shard, tls->mpt_tgt_obj_ult_cnt,
 		tls->mpt_tgt_dkey_ult_cnt, tls->mpt_obj_count, DP_RC(rc));
 free_notls:
+	migrate_obj_put(arg);
```

Comment:
since TLS is pinned by @arg, so I think migrate_obj_put(arg) should be called after migrate_res_release()?